x-pack/legacy/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -397,4 +397,97 @@ describe('Task Runner', () => {
       }
     `);
   });
+
+  test('recovers gracefully when the Alert Task Runner throws an exception when fetching the ApiKey', async () => {
+    encryptedSavedObjectsPlugin.getDecryptedAsInternalUser.mockImplementation(() => {
+      throw new Error('OMG');
+    });
+
+    const taskRunner = new TaskRunner(
+      alertType,
+      mockedTaskInstance,
+      taskRunnerFactoryInitializerParams
+    );
+
+    savedObjectsClient.get.mockResolvedValueOnce(mockedAlertTypeSavedObject);
+
+    const runnerResult = await taskRunner.run();
+
+    expect(runnerResult).toMatchInlineSnapshot(`
+      Object {
+        "runAt": 1970-01-01T00:05:00.000Z,
+        "state": Object {
+          "previousStartedAt": 1970-01-01T00:00:00.000Z,
+          "startedAt": 1969-12-31T23:55:00.000Z,
+        },
+      }
+    `);
+  });
+
+  test('recovers gracefully when the Alert Task Runner throws an exception when fetching Space Level Permissions', async () => {
```

Comment:
I guess the implication is that fetching Space Level Permissions will call the `taskRunnerFactoryInitializerParams.services()` method.  Worth a comment.